onnxruntime/core/providers/rocm/math/matmul_impl.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// Modifications: Remove cudaDeviceProp in LaunchFastGeluKernel.
+// Copyright (c) Advanced Micro Devices, Inc. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/providers/rocm/shared_inc/fpgeneric.h"
```

Comment:
Sort header files in alphabetical order?

Reply:
I see. We can top fpgeneric.h as the first. How about the remaining four header files, can they be sorted?

Reply:
@zhangyaobit Sort the headers is a big no no in ORT, as I mentioned earilier, the headers are not self-contained, and there is an implicit inclusion order requirement if you want them to orchestrate well. Blindly sort them will definitely result in [BV1ct411V73y](https://www.bilibili.com/video/BV1ct411V73y), otherwise.